ext/calendar/cal_unix.c

```diff
@@ -25,14 +25,15 @@
    Convert UNIX timestamp to Julian Day */
 PHP_FUNCTION(unixtojd)
 {
-	time_t ts = 0;
+	time_t ts;
+	zend_bool ts_is_null = 1;
 	struct tm *ta, tmbuf;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|l", &ts) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|l!", &ts, &ts_is_null) == FAILURE) {
```

Comment:
This technically changes behavior if timestamp 0 is passed. But I think the new behavior makes more sense, and we don't document that 0 means current time, so this looks fine to me.